### ISSUE TITLE
Bridge keeps restarting

### ISSUE THREAD

**Issue by Calorion** [kernie66/homebridge-pushcutter#3]
When installed in a HOOBS bridge, Pushcutter keeps restarting, over and over, with these log messages:

```1/28/2022, 12:19:49 PM Pushcutter Bridge Bridge is running on port 51856.
1/28/2022, 12:19:49 PM Pushcutter Bridge Pushcutterhardware: Raspberry Pi 3B+ 1.3 (1GB)
1/28/2022, 12:19:49 PM Pushcutter Bridge Pushcutteros: Raspbian GNU/Linux 10 (buster)
1/28/2022, 12:19:51 PM Pushcutter Bridge Pushcuttergoodbye
1/28/2022, 12:19:58 PM Pushcutter Bridge starting
1/28/2022, 12:20:04 PM Pushcutter Bridge Loaded plugin 'homebridge-pushcutter'
1/28/2022, 12:20:04 PM Pushcutter Bridge Loading 1 platforms...
1/28/2022, 12:20:04 PM Pushcutter Bridge Pushcutterhomebridge-pushcutter v1.1.0, node v16.13.1, homebridge v1.3.9, homebridge-lib v5.2.0
```

**Comment by kernie66** [comment]
I'm sorry to hear that. I don't have HOOBS now, so I haven't tested it on that. 
Is this the first time you tried Pushcutter, or has it worked before this? 
There is not much info to go on, but I can try to update Pushcutter with the latest libs to see if that is the problem. It seems like the libs you are using are newer than what I had when the last version was released.

**Comment by Calorion** [comment]
No, I'm installing Pushcutter for the first time.

**Comment by kernie66** [comment]
I updated the dependencies and released v1.1.1. Please try it to see if HOOBS likes it better.
